### PR TITLE
Logging invalid provided cpp configuration

### PR DESCRIPTION
**Pull request by lygstate**
Signed-off-by: Yonggang Luo <luoyonggang@gmail.com>

### CODE REVIEW

**Review by sean-mcmanus — APPROVED** [review]

File: Extension/src/LanguageServer/client.ts [hunk 1]
```diff
@@ -2511,7 +2511,7 @@ export class DefaultClient implements Client {
 
             sanitized = {...<WorkspaceBrowseConfiguration>config};
             if (!this.isWorkspaceBrowseConfiguration(sanitized)) {
-                console.log("Received an invalid browse configuration from configuration provider.");
+                console.log("Received an invalid browse configuration from configuration provider:" + JSON.stringify(sanitized));
```
Comment (michelleangela) [inline Extension/src/LanguageServer/client.ts]: Please add space after `provider:` on line 2514.

**Review by michelleangela — CHANGES_REQUESTED** [review]

File: Extension/src/LanguageServer/client.ts (hunk 1 above)
Comment (lygstate) [reply Extension/src/LanguageServer/client.ts]: done

**Review by michelleangela — APPROVED** [review]

**Review by michelleangela — APPROVED** [review]

**Review by tranvantam1992 — COMMENTED** [review]
lygstate:main

**Review by tranvantam1992 — COMMENTED** [review]
Extension/src/LanguageServer/client.ts

**Pull request merged**